### PR TITLE
Rename `ScriptResult` to `ScriptCallResult`

### PR DESCRIPTION
**Pull request by sadym-chromium**
#217

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/224.html" title="Last updated on Jun 7, 2022, 2:16 PM UTC (e796693)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/224/bc6aea5...e796693.html" title="Last updated on Jun 7, 2022, 2:16 PM UTC (e796693)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -3747,25 +3747,25 @@ Record=] of type <code>throw</code>, |exception|, is given by:
 [=Remote end definition=] and [=local end definition=]
 
 <pre class="cddl local-cddl remote-cddl">
-ScriptResult = (
-  ScriptResultSuccess //
-  ScriptResultException
+ScriptCallResult = (
```
Comment (jgraham) [inline index.bs:3750]: I wonder if it would be easier to avoid having the typedefs below for each command and just calling this `ScriptEvaluateResult`

Comment (sadym-chromium) [reply index.bs]: Done

File: index.bs [hunk 2]
```diff
@@ -3747,25 +3746,25 @@ Record=] of type <code>throw</code>, |exception|, is given by:
 [=Remote end definition=] and [=local end definition=]
 
 <pre class="cddl local-cddl remote-cddl">
-ScriptResult = (
-  ScriptResultSuccess //
-  ScriptResultException
+ScriptEvaluateResult = (
```
Comment (juliandescottes) [inline index.bs:3749]: The section is called `#### The script.Result type #### {#type-script-Result}`. Should it be renamed to `#### The script.ScriptEvaluateResult type #### {#type-script-ScriptEvaluateResult}` ?

**Review by juliandescottes — COMMENTED** [review]
Overall looks good to me, found two spots which might need updating as well

**Review by juliandescottes — APPROVED** [review]
Thanks a lot for the cleanup! 
I guess we can update the commit to no longer mention `ScriptCallResult`?

**Comment by whimboo** [comment]
Yes, I renamed that part of the commit message before merging. Thanks for the fix!

**Pull request merged**